crc/storage.yaml

```diff
@@ -15,6 +15,8 @@ spec:
     storage: 10Gi
   accessModes:
     - ReadWriteOnce
+    - ReadWriteMany
+    - ReadOnlyMany
```

Comment:
I'm not fully-versed on these `accessModes`, but I wonder if there is a scenario in which it would be desirable to have PVs of `local-storage` storage class with mutually exclusive `accessModes`.  i.e. say, 2 with `RWO`, 2 with `ROX` and 2 with `RWX` (as opposed to having all `accessModes` on every PV)?  @dprince do you have thoughts on this?

Reply:
Yeah, I see your point, and I'm not sure about the potential combination required by different types of operators.
We can try to update the patch and try with a subset of PVs w/ `RWO` and `RWX` (mutually exclusive) and let the scheduler do its job, but in theory, having both `RWX` and `RWO` allows to use the `PV` for both use cases. 
Any particular reason we'd like to have mutually exclusive `accessModes` ?

Reply:
I couldn't think of any off the top of my head, but I just thought I'd throw it out there.  I'm good with the change as-is.